gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -141,7 +145,13 @@ private fun NavGraphBuilder.addAvatarPickerGraph(
         route = QuickEditorPage.EDITOR.name,
         startDestination = EditorNavDestinations.AVATAR_SELECTION.name,
     ) {
-        composable(route = EditorNavDestinations.AVATAR_SELECTION.name) {
+        composable(
+            route = EditorNavDestinations.AVATAR_SELECTION.name,
+            enterTransition = { fadeIn() },
```

Comment:
We could add 
```
popEnterTransition = { fadeIn() + expandVertically() },
```

This would smooth out the transition when coming back from the AltPage, but it should keep the `fadeIn()` only when opening the QE (The `expandVerically()` doesn't work well when the bottom sheet is being expanded). 


https://github.com/user-attachments/assets/7c10091d-7bca-4f0c-a150-a500926ba545

Reply:
TIL -> `popEnterTransition`